common/religion/religions/shamanism_group.txt

```diff
@@ -460,6 +484,14 @@
 			#Main Group
 			doctrine = doctrine_no_head
 
+			#Magic
+			doctrine = doctrine_light_magic_accepted
+			doctrine = doctrine_order_magic_accepted
+			doctrine = doctrine_life_magic_ignored
```

Comment:
Not needed since ignored by the religion.
```suggestion
```

Reply:
Done